dashboard/modules/job/cli.py

```diff
@@ -24,6 +25,10 @@ def _get_sdk_client(
 ) -> JobSubmissionClient:
     client = JobSubmissionClient(address, create_cluster_if_needed, verify=verify)
     client_address = client.get_address()
+    # redact any passwords in the client address
+    secret = re.findall("https?:\/\/.*:(.*)@.*$", client_address)
```

Comment:
Out of curiosity, is the `$` at the end of the regex optional?

Reply:
Good spot, you're correct, it isn't required so I've dropped it in the refactor.